app/MMB/package.json

```diff
@@ -11,6 +11,7 @@
     "@kiwicom/mobile-pdf": "^0",
     "@kiwicom/mobile-relay": "^0",
     "@kiwicom/mobile-shared": "^0",
+    "@kiwicom/react-native-reverse-geocode": "git+ssh://git@github.com:kiwicom/react-native-reverse-geocode.git#master",
```

Comment:
I think we should deploy it to npm and include it from there.

Reply:
But I guess, I really want the guys to have a look at it before publishing it. Or maybe it does not matter?

Reply:
You could always publish it to npm, and if it requires changes, you can publish a new version. But yeah, when @ferrannp and @lukewalczak are happy with the code, you can publish to npm. And then we can merge this pr.